src/sql/src/session/vars.rs

```diff
@@ -1102,6 +1102,31 @@ const ENABLE_STORAGE_SHARD_FINALIZATION: ServerVar<bool> = ServerVar {
     internal: true,
 };
 
+/// Configuration for gRPC client connections.
+mod grpc_client {
+    use super::*;
+
+    pub const CONNECT_TIMEOUT: ServerVar<Duration> = ServerVar {
+        name: UncasedStr::new("grpc_client_connect_timeout"),
+        value: &Duration::from_secs(5),
+        description: "Timeout to apply to initial gRPC client connection establishment.",
+        internal: true,
+    };
+    pub const HTTP2_KEEP_ALIVE_INTERVAL: ServerVar<Duration> = ServerVar {
+        name: UncasedStr::new("grpc_client_http2_keep_alive_interval"),
+        value: &Duration::from_secs(3),
+        description: "Idle time to wait before sending HTTP/2 PINGs to maintain established gRPC client connections.",
+        internal: true,
+    };
+    pub const HTTP2_KEEP_ALIVE_TIMEOUT: ServerVar<Duration> = ServerVar {
+        name: UncasedStr::new("grpc_client_http2_keep_alive_timeout"),
+        value: &Duration::from_secs(5),
+        description:
```

Comment:
curious, is it considered normal for the timeout to be longer than the interval?

Reply:
honestly not sure. I didn't find much for http/2 keep-alive standards, and grpc defaults to not using keep-alives (with baked in defaults of interval=2h, timeout=20s if you turn it on)